docs/en/operations/system_tables.md

```diff
@@ -273,10 +275,68 @@ SELECT * FROM system.metrics LIMIT 10
 │ DistributedSend            │     0 │ Number of connections to remote servers sending data that was INSERTed into Distributed tables. Both synchronous and asynchronous mode.                                                          │
 └────────────────────────────┴───────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+
 **See Also**
 
 - [system.asynchronous_metrics](#system_tables-asynchronous_metrics) — Contains periodically calculated metrics.
 - [system.events](#system_tables-events) — Contains a number of events that occurred.
+- [system.metric_log](#system_tables-metric_log) — Contains a history of metrics values from tables `system.metrics` и `system.events`.
+- [Monitoring](monitoring.md) — Base concepts of ClickHouse monitoring.
+
+## system.metric_log {#system_tables-metric_log}
+
+Contains history of metrics values from tables `system.metrics` and `system.events`, periodically flushed to disk.
+For turn on metrics history collection on `system.metric_log`, create `/etc/clickhouse-server/config.d/metric_log.xml` with following content:
+```xml
+<yandex>
+    <metric_log>
+        <database>system</database>
+        <table>metric_log</table>
+        <flush_interval_milliseconds>7500</flush_interval_milliseconds>
+        <collect_interval_milliseconds>1000</collect_interval_milliseconds>
```

Comment:
Probably there should be TTL or some depth in this table configuration or size would grow without a limit

Reply:
@qoega 
I agree, could you suggest code changes in 
https://github.com/ClickHouse/clickhouse/tree/master/dbms/src/Interpreters/MetricLog.h
https://github.com/ClickHouse/clickhouse/tree/master/dbms/src/Interpreters/MetricLog.cpp
for creating MergeTree table with TTL?

Reply:
as i see in https://github.com/ClickHouse/ClickHouse/blob/master/dbms/src/Interpreters/SystemLog.cpp#L27

currently, we don't have the ability to use TTL for system tables